packages/components/src/ListView/Items/Item/Item.component.js

```diff
@@ -1,14 +1,16 @@
 import React, { Component } from 'react';
 import classNames from 'classnames';
+import { translate } from 'react-i18next';
 import theme from './Item.scss';
 import ItemPropTypes from './Item.propTypes';
 import Action from '../../../Actions/Action';
+import I18N_DOMAIN_COMPONENTS from '../../../constants';
 
 function itemLabelClasses() {
 	return classNames(theme['tc-listview-item-label'], 'tc-listview-item-label');
 }
 
-class Item extends Component {
+export class ItemComponent extends Component {
```

Comment:
Just a heads-up, Was this added to be imported by the test file? It's still importing the default

Reply:
What's the convention for the components suffix when wrapped or will be wrapped ?

Reply:
@jdescottes-talend there is no real convention but in practice, to stay in line with the existing code, when you need to export a component that is not the default export you add the `Component` suffix. Changing the name helps user to avoid mistake in their imports.